src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/Pages/FetchData.razor

```diff
@@ -38,14 +38,14 @@ else
 }
 
 @code {
-    private WeatherForecast[] forecasts;
+    private WeatherForecast[] _forecasts;
```

Comment:
The ones above I'm 😐 about since they're sample. However, going by my discussion, we would use `this.` to disambiguate locals versus global, and not prefix items with underscores.

Reply:
`forecasts` (here) didn't use `this` before. You're saying that it *should have* used `this` in the first place?

> disambiguate locals versus global

I vaguely recall talking to Taylor about this a long time ago. I'd need to dig up the convo and refresh myself on what he was saying about it. The convo pertained to Razor expressions `@{ ... }` versus code blocks `@code { ... }`. I'll see if I can track that down tomorrow.

My concern is that there's a discrepancy between the way the docs are showing similar scenarios and the templates. IIRC, the ones that didn't have an underscore (in engineering repo tests and perhaps samples/examples/posts/talks) were the ones inside of a **class** in the `@code` block. I thought that others had underscores.

Whatever the case over here, that's the way that the docs are set up. Blazor doc examples never (or very rarely) use `this`. All of the `@code` block fields use underscores. On review, DR, SS, and RN *seemed* ok with what the doc examples are doing. However, this inconsistency between templates and the docs is present. It seems like it would be confusing to devs.

I hope it can be addressed on one side or the other for consistency.